pubsub/gossipsub/gossipsub-v1.1.md

```diff
@@ -134,6 +135,29 @@ message PeerInfo {
 }
 ```
 
+### Signature policy
+
+The usage of the `signature`, `key`, `from`, and `seqno` fields in `Message` is now configurable.
+These fields may negatively affect privacy in content-addressed messaging,
+and may need to be strictly enforced in author-addressed messaging.
```

Comment:
I think the content-addressed and author-addressed word choice might be a bit confusing. How about "anonymous" and "origin-stamped"?